internal/operators/odf/odf_operator.go

```diff
@@ -161,7 +161,7 @@ func (o *operator) GetHostRequirements(_ context.Context, cluster *common.Cluste
 	}
 
 	role := common.GetEffectiveRole(host)
-	if numOfHosts <= 3 { // Compact Mode
+	if numOfHosts <= 3 || numOfHosts <= 5 { // Compact Mode
```

Comment:
number of hosts should be greater than or equal to three and less than or equal to 5 for compact mode